### PR TITLE
setting up Hello microverse project

### PR DESCRIPTION
**Pull request by gasangw**
****Steps to accomplish the project**.**
Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
Create a new GitHub repo.
Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
Add a .gitignore file.
Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
Add 3 files (each one in a separate commit):
A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
A .css file that includes one class with styles for your header (make it in your favorite color).
A test.md file with the text "This file should be ignored by git".
Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
Open a pull request.

### CODE REVIEW

File: .github/workflows/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```
Comment (AnselemOdims) [inline .github/workflows/.gitignore]: - [x] Kindly remove the `.gitignore` fom the workflows folder and place it in the root folder

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @gasangw, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:

## Required Changes ♻️
- [ ] Please make the linters are all passing their checks. At the moment, some are failing
- [ ] There is no `.css` file. Kindly include a `.css` file
- [ ] Kindly check the in-line comments below

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .github/workflows/test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (RolandM99) [inline .github/workflows/test.md:1]: - [ ] Kindly, remove the `test.md` file in the `.github/workflows` folder.

**Review by RolandM99 — COMMENTED** [review]
Hi @gasangw , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._
- [ ] As requested by the previous reviewer, your project should have a `.css` file, Kindly add.
- [x] you have to add the `test.md` file in your `.gitignore` file. kindly add that file. The previous reviewer requested you to remove that  file in the `.github` directory not to delete it from your project.
- [ ] You're almost there, But up now you didn't yet setup very well linters. make sure that you setup the linters following this [documentation](https://github.com/microverseinc/linters-config/tree/master/html-css#troubleshooting)
- [ ] You have to close a PR after a code reviewer approve it, Kindly reopen this PR.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gasangw** [comment]
Thanks for the review I added the .css file but it's on the folder of
styling as well as the test.md file. All changes are in the styling branch.
I wanted to request a git pull but when I saw the information of using the
previous one so I didn't that's why I think some of my changes you aren't
seeing them

Thank you for the feedback.

**Pull request closed without merging**